libs/pubsub/query/query.go

```diff
@@ -19,6 +19,8 @@ import (
 
 var (
 	numRegex = regexp.MustCompile(`([0-9\.]+)`)
+
+	_ = "test"
```

Comment:
wait, what?

Reply:
this is just a spurious change to make the CI do some work, I'll revert it before merging.

Reply:
ah yup, figured that out when I saw the other PR